pom.xml

```diff
@@ -47,7 +47,7 @@
 
         <deps.apache-sshd.version>0.12.0</deps.apache-sshd.version>
         <deps.jsch.version>0.1.52</deps.jsch.version>
-        <deps.ssh-credentials-plugin.version>1.13-SNAPSHOT</deps.ssh-credentials-plugin.version>
+        <deps.ssh-credentials-plugin.version>2.0-SNAPSHOT</deps.ssh-credentials-plugin.version>
```

Comment:
I am just curious why you say this is an anti-pattern. Can you enlighten me? 

I have followed this practice for quite a while in quite a few projects and have not had any ill side affects and it makes it easy for me to find all the dependency versions used in a project.